src/backends/tensorrt/plan_backend.cc

```diff
@@ -111,6 +111,24 @@ PlanBackend::Context::~Context()
   }
 }
 
+bool
```

Comment:
Use ConvertDataType from plan_utils.cc

Reply:
plan_utils.cc contains a function `DataType ConvertDatatype(nvinfer1::DataType trt_type)` but not a function to test convert the DataType in config file to a nvinfer1::DataType. 
I can move this function to the plan_utils.cc and create the appropriate fucntion: `nvinfer1::DataType ConvertDatatype(DataType dtype)`

Reply:
Yes, replace this with a new ConvertDatatype in plan_utils.